### PR TITLE
Documentation/docstrings D

### PR DESCRIPTION
**Pull request by santiagoinfantinom**
Adresses #110

### CODE REVIEW

File: src/D1_economic_functions.py [hunk 1]
```diff
@@ -1,21 +1,77 @@
 import logging
 
-# annuity factor to calculate present value of cash flows
+
 def annuity_factor(project_life, wacc):
+    """
+    Annuity factor to calculate present value of cash flows (?)
```
Comment (santiagoinfantinom) [inline src/D1_economic_functions.py:6]: I still don't fully understand the concept of this annuity_factor. Can you take a look at this?

File: src/D1_economic_functions.py [hunk 2]
```diff
@@ -1,21 +1,77 @@
...
 
-# accounting factor to translate present value to annual cash flows
+
 def crf(project_life, wacc):
+    """
+    Accounting factor to translate present value to annual cash flows
+
+    Parameters
+    ----------
+    project_life: int
```
Comment (santiagoinfantinom) [inline src/D1_economic_functions.py:33]: There's a project_life and a project_lifetime and both are ints. What do they represent exactly?

File: src/D1_economic_functions.py [hunk 3]
```diff
@@ -42,6 +98,26 @@ def present_value_of_changing_fuel_price(
...
+    investment_t0: float
+
+    lifetime:int
+
+    project_life:int
+
+    wacc: float
+        Discount factor
+
+    tax:int
```
Comment (santiagoinfantinom) [inline src/D1_economic_functions.py:114]: What exactly is this tax here?

File: src/D1_economic_functions.py [hunk 4]
```diff
@@ -75,10 +151,41 @@ def capex_from_investment(investment_t0, lifetime, project_life, wacc, tax):
 
 
 def annuity(present_value, crf):
+    """
+    Calculates the regular payment including interests(?)
```
Comment (santiagoinfantinom) [inline src/D1_economic_functions.py:155]: Here taking a leap trying to figure out what this annuity actually is..

**Review by santiagoinfantinom — COMMENTED** [review]
Hi @smartie2076, I did D0 and D1 in the same PR. Most of my questions are related to D1 as it uses a lot of economics terminology which is not quite clear to me (wacc, cfr, annuities). Could you please take a look at my comments? I will the correct the docstrings

File: src/D1_economic_functions.py (hunk 4 above)
Comment (smartie2076) [reply src/D1_economic_functions.py:155]:
```suggestion
    Calculates the constant annual payments (annuities) from a present value
```

**Review by smartie2076 — COMMENTED** [review]
For D1, please reference the definitons here: https://github.com/rl-institut/mvs_eland/blob/dev/src/mvs_eland/C2_economic_functions.py

**Pull request merged**